float32[] linear_vel
float32[] angular_vel
float32[] joint_angles
float32[] joint_velocities
bool[] toes_contact
bool[] thighs_contact
bool[] shanks_contact
